lightning/tests/lightning_config_max_error/run.sh

```diff
@@ -28,7 +28,7 @@ duplicated_row_count=$(( ${total_row_count} - ${uniq_row_count} ))
 remaining_row_count=$(( ${uniq_row_count} + ${duplicated_row_count}/2 ))
 
 run_sql 'DROP TABLE IF EXISTS mytest.testtbl'
-run_sql 'DROP TABLE IF EXISTS lightning_task_info.conflict_error_v3'
+run_sql 'DROP TABLE IF EXISTS lightning_task_info.conflict_error_v4'
```

Comment:
Here we changed name of table，will this result we possible leave lightning_task_info.conflict_error_v3' table can not be drop forever

Reply:
Yes. This is integration test script to cleanup old data. But in real cases user may see old tables. Just as the old behaviour, lightning will not automatically drop them.

Reply:
Can we also enable drop old version table and create new version one to avoid the old tables are still there.